clang/unittests/Format/FormatTestComments.cpp

```diff
@@ -182,7 +182,7 @@ TEST_F(FormatTestComments, UnderstandsSingleLineComments) {
                    "int   a;     // This is unrelated"));
   EXPECT_EQ("class C {\n"
             "  void f() { // This does something ..\n"
-            "  }          // awesome..\n"
+            "  } // awesome..\n"
```

Comment:
But this would work against a conscious decision. Should we change that? Or should we add `TT_FunctionRBrace` (shouldn't be hard) and exempt them from not aligning?

Reply:
I think the comment at the bottom should _not_ be aligned with the one at the top, so we are fine here.

Reply:
Your call @mydeveloperday, I also think we should not align.

Reply:
How long do we wait here?

Reply:
I think it's been long enough.